keps/sig-node/1967-size-memory-backed-volumes/kep.yaml

```diff
@@ -20,13 +21,13 @@ stage: beta
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.22"
+latest-milestone: "v1.32"
```

Comment:
Do we think anyone has become dependent on keeping the feature off over these 10 releases?

Reply:
The feature has been on in beta. So I don’t think so.